libraries/AP_Logger/LogStructure.h

```diff
@@ -1385,6 +1400,19 @@ struct PACKED log_PSCZ {
 // @Field: CAn: Angle to closest object
 // @Field: CDis: Distance to closest object
 
+// @LoggerMessage: PRXR
+// @Description: Proximity Raw sensor data
+// @Field: TimeUS: Time since system startup
+// @Field: Layer: Pitch(instance) at which the obstacle is at. Each layer is 30 degrees wide, 0th layer is from - 75 degrees to -45. 2nd layer captures horizontal data.
```

Comment:
This is a little unclear.

I wonder if we should put the actual degrees in instead.

```suggestion
// @Field: Layer: Pitch(instance) at which the set of measurements is at. By default each layer is 30 degrees wide, 0th layer is from - 75 degrees to -45. 2nd layer captures horizontal data.
```